android/src/main/java/tv/vizbee/rnsender/VizbeeBootstrap.java

```diff
@@ -74,4 +76,11 @@ public void initialize(Application applicationContext, String vizbeeAppId, Vizbe
       VizbeeContext.getInstance().init(applicationContext, vizbeeAppId, new VizbeeAppAdapter(), options);
   }
 
+   //----------------
+    // Set Theme
+    //----------------
+
+    public void setAppTheme(@StyleRes int theme) {
+        VizbeeContext.getInstance().setTheme(theme);
+    }
```

Comment:
Revert this change if it's only used in the demo app. Apps can directly call  
`VizbeeContext.getInstance().setTheme(theme);` in their java files. Add `vizbee-sender-sdk` as a dependency to the apps module if necessary.

Reply:
Apps have to add the dependency of native SDK

Reply:
ok got it